src/containers/NoteList.tsx

```diff
@@ -115,6 +115,7 @@ const NoteList: React.FC = () => {
               >
                 <MoreHorizontal size={15} />
               </div>
+              {note.favorite && <span>⭐</span>}
```

Comment:
Hey!
We use the amazing Feather icons 🐱‍🏍

I think that would be pretty cool if we use the same icon that is used in the SideBar!
(if am I right, I think that is `Bookmark`)

Reply:
I should have considered this, instead I rushed. I have updated the PR with the correct icon!